node/src/parachain/dev_chain_spec.rs

```diff
@@ -143,12 +143,14 @@ fn configure_genesis(
 		block_reward: BlockRewardConfig {
 			// Make sure sum is 100
 			reward_config: pallet_block_reward::RewardDistributionConfig {
-				treasury_percent: Perbill::from_percent(20),
-				dapps_percent: Perbill::from_percent(25),
-				collators_percent: Perbill::from_percent(10),
-				lp_percent: Perbill::from_percent(25),
-				machines_percent: Perbill::from_percent(10),
-				parachain_lease_fund_percent: Perbill::from_percent(10),
+				treasury_percent: Perbill::from_percent(25),
+				dapps_percent: Perbill::from_percent(0),
+				collators_percent: Perbill::from_percent(40),
+				lp_percent: Perbill::from_percent(10),
```

Comment:
Please change the naming because it'll be easier for people to read. However, please check whether we have to write the migration or not

Reply:
Noted i will change the naming. Regarding the migration i'm not sure if it will be necessary since i already added the code for the migration of the values. Changing the name will be handle there as well.